test/smoke.bats

```diff
@@ -113,3 +113,17 @@ setup() {
 
   [ ! -f input.txt ]
 }
+
+@test "it reads from stdin" {
+  echo "I would like to encrypt this!" > input.txt
+
+  cryptdo-bootstrap --passphrase "password" input.txt
+  rm input.txt
+
+  run bash -c 'echo "New encrypted stuff" | cryptdo --passphrase "password" -- bash -c "cat - > input.txt"'
+  [ "$status" -eq 0 ]
```

Comment:
I don't think this will ever fail because neither of the `bash` invocations in the previous line have the `-e` flag.